pcap.js

```diff
@@ -13,12 +13,39 @@ exports.TCPSession = tcp_tracker.TCPSession;
 exports.DNSCache = DNSCache;
 
 function PcapSession(is_live, device_name, filter, buffer_size, outfile, is_monitor) {
+    var self = this;
```

Comment:
I moved the self var back to its original location.
Also, if that's not the issue of style that you're referring to, would you mind expounding on that?
If you're referring to the use of an object as an argument, I can supply a large list of libraries within node itself that use this construct.

I felt that it was an improvement, in both usability and readability, on the alternative of just adding another parameter and calling the function like this:
`var pcapSession = pcap.createSession(interface, "", null, null, 500);`